archive/zip_archiver.go

```diff
@@ -47,7 +47,24 @@ func (a *ZipArchiver) ArchiveFile(infilename string) error {
 		return err
 	}
 
-	return a.ArchiveContent(content, fi.Name())
+	if err := a.open(); err != nil {
+		return err
+	}
+	defer a.close()
+
+	fh, err := zip.FileInfoHeader(fi)
+	if err != nil {
+		return fmt.Errorf("error creating file header: %s", err)
+	}
+	fh.Name = fi.Name()
+
```

Comment:
Can you set the method to deflate here per the issue?

Reply:
Sure, I'll make the same change in `ArchiveDir` as well.

Reply:
I am hoping it will fix the issue, but it's honestly just my best guess since I don't know how to reproduce the issue that was reported.